diff --git a/lib/util.js b/lib/util.js
index 709795bd8cbeec83460d523e478adaf259788c94..d83c688047cf73892b4398826d4984804a8d3b4f 100755
--- a/lib/util.js
+++ b/lib/util.js
@@ -274,7 +274,7 @@ const setPropInsensitive = (obj, prop, value) => {
 };
 
 const dumpDir = PATH.resolve(__dirname, '../dumps/');
-if (FS.existsSync(dumpDir)) FS.rmdirSync(dumpDir, { recursive: true });
+if (FS.existsSync(dumpDir)) FS.rmSync(dumpDir, { recursive: true, force: true });
 
 exports.logger = content => {
   const file = PATH.resolve(dumpDir, `${Date.now()}-${Math.random().toString(36).slice(3)}.txt`);
